lib/perl/Genome/Model/Build/Command/Start.pm

```diff
@@ -143,6 +143,9 @@ sub create_and_start_build {
 
     my $outer_transaction = UR::Context::Transaction->begin();
     $model->build_requested(0);
+    my $anp = $model->analysis_project;
+    my $config_dir = $anp? $anp->environment_config_dir : undef;
```

Comment:
I guess this check is in here to hedge our bets - but it seems like people shouldn't be starting builds that aren't linked to analysis projects anyways. Would it make sense to `die` if an AnP is not present?